libs/utils/utils.js

```diff
@@ -104,6 +105,17 @@ export async function loadBlock(block) {
   return block;
 }
 
+export async function loadLCP({ blocks, lcpList, loader }) {
+  const list = lcpList || LCP_BLOCKS;
+  const blockLoader = loader || loadBlock;
+  const lcpBlock = blocks.find((block) => list.includes(block.classList[0]));
```

Comment:
Will this work if there are multiple LCP blocks?  Should it be `blocks.filter` and then loop through them?

Reply:
Yes, but to clarify: It will only find the first. This is by design. `blocks` is a top to bottom list of blocks on the page. So the first hit is what is returned. We do not want multiples.